### ISSUE TITLE
share/examples/pocketcoin.conf is horribly out of date

### ISSUE THREAD

**Issue by the-real-vortex-v** [pocketnetteam/pocketnet.core#655]
**Describe the bug**
share/examples/pocketcoin.conf is really out of date. The docker config is more up to date than the actual config example in the generic pocketcoin.conf.
There have been some questions about what the extra new ports do and which ones people should allow through their firewall. I found them listed in the docker config files (in the readme.md) but they are not in the examples for any other install options such as windows. I strongly suggest when some one does something like this that they update the documentation for every OS to avoid confusion in the future. Even one line that points to the docker config (so people can find an explination of the ports) and then the pocketcoin.conf gets updated with any new information before a release would be a good idea.
The docker file that has the actual committed explanation:
https://github.com/pocketnetteam/pocketnet.core/commit/7f075c4b6c8914ee5d88eae1359da8a23b62c44a
these are the readme.md files. The port information is just not shared with the windows section. This is confusing to the average user who expects all the relevant information to be shared under each OS separately. It's not a clean way to explain things.

**Expected behavior**
Documentation to be consistent and have changes listed in the correct places. Even if it is just a reference to where it's explained. In this case the full list of ports that should be in the config are in a docker file and some of them are not mentioned in the pocketcoin.conf for the seperate OSes. They should be listed under each and explained WHY they need to be included and what should be firewalled etc.

**Desktop (please complete the following information):**
All OS. All Browser etc. This is a config/documentation issue.

**Additional context**
TLDR; This issue is going to make it difficult for the average younger audience to understand. If you've been using unix/linux etc for more than 20 years then this isn't an issue. If you are new to linux/docker/MAC/Windows etc then this will be massively confusing as people are taught that the different OSes are not the same so they need to only follow their steps for the OS they are using. Having the information spread out over more than one file and place/OS is just a recipe for confusion and bad form.
It may seem like a minor issue to some but it's a big issue for new users. If the project wants to attract more uptake then streamlining this stuff should be a priority right now. It's also not a hard thing to do. Just have some one check that documentation is updated in the pocketcoin.conf and the readme.md etc.

**Comment by andyoknen** [comment]
Thank you for the offer. Could you help with the paperwork? Where and in what form are corrections needed? Maybe a Pull Request?

Here is the purpose of all the ports used:
- 37070 : The main port for communication between nodes
- 37071 : Service port (Private RPC) for node management (it is not recommended to open in the firewall)
- 38081 : Public RPC port for API implementation
- 38881 : The same as 38081, but for HTTPS (a self-signed certificate is used)
- 8087 : A public WebSocket port for implementing notification mechanisms in the browser and application (not push notifications)
- 8887 : The same as 8087, but for WSS (a self-signed certificate is used)

**Comment by the-real-vortex-v** [comment]
I told you. The pocketnet.conf needs to be updated and there should be only one version that is shared between the different installation methods (docker vs native windows/linux etc software). I'm not about to do this as I also said I have no idea what the ports are used for. Also you just provided the information in your last post. Why not just add that to the readme/pocketnet.conf instead? 
Whoever is adding in the features needs to also add documentation. Even a short note in the readme.md and/or man page to explain this stuff is fine.

So TLDR; update the pocketnet.conf with this info. Have only one readme.md and pocketnet.conf file (rather than multiple) that has sections as required that explain any changes with the duplicated information removed. This is fairly standard stuff that you can learn from reading most manuals that have windows/linux/mac etc ports.

As a future fun project someone could add preprocessor "macros" to the config file that evaluates simple if statements for things like arch or OS_version (and expands/removes stuff from the config) and that way you could have a docker/windows/linux config if there is a requirement to do so. I'm not a c/c++ coder so I'm not exactly in a position to contribute to this. Nor am I a professional coder. My coding is horrible and self taught. I don't know about refactoring or modern fad languages. I barely know what I know.

**Comment by andyoknen** [comment]
The sample configuration file needs to be finalized
